include/util.h

```diff
@@ -84,4 +84,6 @@ static inline int cnc_strdup(char **string, char *string_to_dup)
 	return 0;
 }
 
+void construct_dump_path(char *path, char *file_format);
```

Comment:
Let's add some documentation.

Also, where does the `file_format` argument name comes from? Shouldn't it be `filename`?

Reply:
I named it `file_format` because different databases use different file formats to store the `dump` file. For example postgres uses `.dump` and mongo uses `.bson`. But `filename` works too.

Reply:
But it is used for the filename, right?

Reply:
Yeah the file name doesnt matter all that much since its a generic name/placeholder.
But ill changed it to filename since it more accurately represents what the argument is.